src/server/pachyderm_test.go

```diff
@@ -7142,7 +7158,9 @@ func TestSkippedDatums(t *testing.T) {
 }
 
 func TestOpencvDemo(t *testing.T) {
-	t.Skip("flaky")
```

Comment:
I think we may actually need to leave this in, so the test doesn't start running in Travis

Reply:
oh, I thought I added this skip as part of this branch! That's odd.